### ISSUE TITLE
Saving failed: Feature image alt text cannot be longer than 125 characters.

### ISSUE THREAD

**Issue by mhansen** [TryGhost/Ghost#13421]
### Issue Summary

Ghost(Pro) on Chrome.

### To Reproduce

1. Add a feature image
2. Add alt text, e.g. for me "Profilers measure the OS/Hardware/Runtime, outputting Profile Data Formats. Converters can convert one Profile Data Format to another. Analysis UIs read Profile Data Formats."

![image](https://user-images.githubusercontent.com/105529/135700972-974be781-66b1-4773-a77e-e7303b07053a.png)

Expected: be able to spell out the content of the image for vision-impaired users.

Actual: arbitrary limitation of 125characters for alt text; this is not enough to convey the depth of the diagram I'm trying to post. Vision-impaired users will be unable to understand this.

Fix: remove this validation check. I don't think there's a good reason to have it, and it disadvantages blind readers. I don't think this is what you intended. For some prior art, Twitter allows 1000 characters, but even that should probably not be limited.

![image](https://user-images.githubusercontent.com/105529/135700963-f7f6dcfe-e031-47ba-8293-bc4b3b1c310c.png)

### Technical details:

* Ghost Version: Whatever's on Ghost(pro) right now
* Browser/OS: Chrome on macOS

**Comment by ErisDS** [comment]
Reference material for the 125 character limit: https://moz.com/learn/seo/alt-text
Full discussion can be seen here: https://github.com/TryGhost/Ghost/pull/13030#pullrequestreview-680572215

We've followed accessibility guidelines and best practice here to the best of our ability. 

I'm going to close as this is not a bug. We carefully consider data limitations before implementing them, not least because with our decentralised setup we have a strict policy on not running column alteration migrations outside a major version.

**Comment by mhansen** [comment]
While many sites reference a 100/125 character limit, I think they're based on ancient folklore that no longer applies in 2021. None of them really cite their sources, except a few that reference JAWS 6.0.

e.g. JAWS 6.0, a 16-year-old version of the screen reader, is documented as having this 125-character limit: https://terrillthompson.com/tests/altlength.html. 

Current versions of screen readers have no limits on the size of the alt text they'll read: https://www.washington.edu/doit/how-long-can-alt-attribute-be

It seems that a more modern JAWS version reads 150 characters (by default), then asks the user if they'd like to continue the reading, or scroll past.

I think a few of the older references wanted you to use `longdesc` for long `alt` descriptions, but that got deprecated in HTML5: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/longDesc, so I think longer alt attributes are the way to go now.

**Comment by stuartpb** [comment]
For comparison, the maximum length for alt text on Twitter is 1000 characters: https://help.twitter.com/en/using-twitter/picture-descriptions

**Comment by mhansen** [comment]
If you're going to close this as not-a-bug, could I make a feature request to enforce the text limit size in the client's textfield so we don't have to backspace some number of characters and retry saving, getting an error message each time, looping until it works?

(to be clear, my preferred solution would be: support longer alt-text; I just hit this issue again today trying to explain a graphic in alt text and failed, so unsighted readers just won't get the full context! it's a shame that tool limitations are getting in the way of accessibility)

**Comment by zeitkunst** [comment]
I'd like to re-open this as I consider it a bug that has not been resolved. Currently using Ghost 5.100.1 self-hosted. It's 2024 and alt text longer than 125 characters is standard across major platforms such as Mastodon, Bluesky, and X/Twitter. Sometimes images of text (i.e., of newspapers, print articles, screenshots, etc) are necessary, and 125 character limit does not allow these images to be properly described. Not everything is born-digital, and not everything has a URI that can be linked to, and thus sometimes images of print materials will still be needed online and therefore need to be described in a screen-reader-capable manner--and this requires more than 125 characters.

If Ghost does not do it already, I would encourage them to work with visually-impaired users to get their feedback on this.
